Formula/lensfun.rb

```diff
@@ -17,11 +19,14 @@ class Lensfun < Formula
   depends_on "gettext"
   depends_on "glib"
   depends_on "libpng"
-  depends_on "python"
+  depends_on "python@3.8"
 
   def install
     system "cmake", ".", *std_cmake_args
     system "make", "install"
+
+    rewrite_shebang detected_python_shebang,
+      "lensfun-add-adapter", "lensfun-convert-lcp", "lensfun-update-data"
```

Comment:
```suggestion
    rewrite_shebang detected_python_shebang,
      bin/"lensfun-add-adapter", bin/"lensfun-convert-lcp", bin/"lensfun-update-data"
```

Or alternatively, we can use a virtualenv instead (unless the `lensfun` Python package is more than just an internal thing and is used publicly).